skafka/src/main/scala/com/evolutiongaming/skafka/producer/Producer.scala

```diff
@@ -113,15 +112,14 @@ object Producer {
     fromProducerJ1(producer)
   }
 
-
   private sealed abstract class Main
 
   @deprecated("use `fromProducerJ1` instead", "11.7.0")
-  def fromProducerJ[F[_]: Effect: Blocking](producer: F[ProducerJ[Bytes, Bytes]]): Resource[F, Producer[F]] = {
+  def fromProducerJ[F[_]: Blocking: Sync: Async](producer: F[ProducerJ[Bytes, Bytes]]): Resource[F, Producer[F]] = {
```

Comment:
do we need both `Sync` & `Async` separately here?

Reply:
I think we should delete deprecated methods and not try to migrate them to CE3

Reply:
Removed deprecated methods